PublicationsRouter/v2/sword-out/pubrouter-xml/pubrouter.xsd

```diff
@@ -0,0 +1,285 @@
+                <xs:element ref="dcterms:dateAccepted" minOccurs="0"/>
+                <xs:element ref="rioxxterms:publication_date" minOccurs="0"/>
+                <xs:element name="history_date" minOccurs="0" maxOccurs="unbounded">
+                    <xs:annotation>
+                        <xs:documentation xml:lang="en">
+                            Dates of historical changes to the publication outside of the publication date
+                            (date of accepted manuscript, etc)
+                        </xs:documentation>
+                    </xs:annotation>
+                    <xs:complexType>
+                        <xs:simpleContent>
+                            <xs:extension base="xs:date">
+                                <xs:attribute name="type" type="xs:string" use="required"/>
+                            </xs:extension>
+                        </xs:simpleContent>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element ref="rioxxterms:project" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:element ref="ali:free_to_read" minOccurs="0"/>
+                <xs:element name="license" minOccurs="0" maxOccurs="unbounded">
+                    <xs:annotation>
+                        <xs:documentation xml:lang="en">
+                            License information about this publication
+                        </xs:documentation>
+                    </xs:annotation>
+                    <xs:complexType>
+                        <xs:simpleContent>
+                            <xs:extension base="xs:string">
+                                <xs:attribute name="start_date" type="xs:date"/>
+                                <xs:attribute name="url" type="xs:string"/>
+                                <xs:attribute name="version" type="xs:string"/>
+                            </xs:extension>
+                        </xs:simpleContent>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="embargo" minOccurs="0">
+                    <xs:annotation>
+                        <xs:documentation xml:lang="en">
+                            When an embargo starts or ends for this publication
+                        </xs:documentation>
+                    </xs:annotation>
+                    <xs:complexType>
+                        <xs:attribute name="start_date" type="xs:date"/>
+                        <xs:attribute name="end_date" type="xs:date"/>
+                        <xs:assert test="(@start_date or @end_date)"/>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="author" minOccurs="0" maxOccurs="unbounded" type="contributor_type">
+                    <xs:annotation>
+                        <xs:documentation xml:lang="en">
+                            Stores information about the author, like email, ORCID, type.
```

Comment:
Better as "Author details, including name and other information where available such as email, ORCID etc"